app/tools/android/make_apk.py

```diff
@@ -690,6 +691,13 @@ def MakeEmbeddedApk(options, app_info, app_dir, packaged_archs):
       sys.exit(13)
 
 
+@atexit.register
+def PrintDeprecationMessage():
+    print('\n\n\n   WARNING!!! This command is deprecated and will be removed soon.\n\n'
+          '   Please use crosswalk-app-tools instead. For more information see\n'
+          '   https://crosswalk-project.org\n')
```

Comment:
We should probably point to a more precise link saying why we are doing this and what the deprecation plan is.